src/main/resources/test_files/runtime-spark/DataFrames/LetClauseType2.jq

```diff
@@ -0,0 +1,3 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:21:" :)
+let $y as integer := parallelize((1, 2))
+return $y
```

Comment:
This file should be named as 'LetClauseTypeCheckError[number].jq'

Reply:
DONE